Makefile

```diff
@@ -0,0 +1,54 @@
+all: run
+
```

Comment:
(this is just for me to be able to more easily run this without having `web` running in a container 😅 )

```
make database
make worker
make run
```

(I should've named `run` `web`).